docs/installing.rst

```diff
@@ -19,7 +19,7 @@ Simulators:
    -  `Aldec Active-HDL`_: Tested with Active-HDL 9.3, 10.1, 10.2, 10.3 (x64/x86)
    -  `GHDL`_
 
-      -  Works with versions >= 0.33
+      -  Works with versions >= 0.33 (Note: GHDL is a rolling project, it is therefore recommended to use the latest `nightly release`_ tarball.)
```

Comment:
Please, write the URL inline:

```rst
`nightly release <https://github.com/ghdl/ghdl/releases/tag/nightly>`_
```

Also, you can write the Note as an admonition:

```rst
      -  Works with versions >= 0.33
         .. NOTE:: GHDL is a rolling project, it is therefore recommended to use the latest `nightly release <https://github.com/ghdl/ghdl/releases/tag/nightly>`_ tarball.
```

Reply:
Inlined the URL, as you suggested.